translate/storage/jsonl10n.py

```diff
@@ -409,6 +409,122 @@ def _extract_units(
             yield from parent
 
 
+I18NextV4Suffixes = ["zero", "one", "two", "few", "many", "other"]
```

Comment:
I'd use `translate.lang.data.cldr_plural_categories` instead of redefining it here, it is based on CLDR anyway.